prod/westeurope/internal/api/functions_admin_r3/web_test/terragrunt.hcl

```diff
@@ -0,0 +1,34 @@
+# Common
+dependency "application_insights" {
+  config_path = "../../../../common/application_insights"
+}
+
+dependency "resource_group" {
+  config_path = "../../../../common/resource_group"
+}
+
+dependency "function_app" {
+  config_path = "../function_app"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_application_insights_web_test?ref=v2.1.34"
+}
+
+inputs = {
+  name = "func-admin"
+
+  resource_group_name     = dependency.resource_group.outputs.resource_name
+  application_insights_id = dependency.application_insights.outputs.id
+  # test disabled: so far the info page requires the authorization key we don't want to put in web tests.
```

Comment:
@pasqualedevita .. function admin ... same "issue" of function backoffice: the info api requires the authorization key :-(

Reply:
auth level it's ok
https://github.com/pagopa/io-functions-admin/blob/master/Info/function.json
I tried to manual enable it and it works. I think that we can enable it

Reply:
thanks @pasqualedevita I am going to enable it.